module/event-sourcing/src/Application/Middleware/DbalTransactionMiddleware.php

```diff
@@ -6,7 +6,7 @@
 
 declare(strict_types=1);
 
-namespace Ergonode\EventSourcing\Infrastructure\Middleware;
+namespace Ergonode\EventSourcing\Application\Middleware;
```

Comment:
It shouldn't be changed. It's adding database transaction which is part of the infrastructure responsibility I think